test/tests.cpp

```diff
@@ -174,7 +174,7 @@ TEST_CASE("test_qengine_cpu_par_for_skip_wide")
     QEngineCPUPtr qengine = std::make_shared<QEngineCPU>(1, 0);
 
     const int NUM_ENTRIES = 2000;
-    const int NUM_CALLS = 1000;
+    // const int NUM_CALLS = 1000;
```

Comment:
Dead line?

Reply:
Magically, when I replaced the Catch2 header, it turned out these weren't used ever, so they suddenly started throwing warnings, promoted to errors. I'll remove them.

Reply:
Removed.